feature/gnoi/factory_reset/factory_reset_test.go

```diff
@@ -0,0 +1,206 @@
+package factory_reset
```

Comment:
ideally, we should not have underscores in package names. Can you update?
https://go.dev/doc/effective_go#package-names